content/docs/getting-started/adding-a-catalog-item/github/index.md

```diff
@@ -0,0 +1,121 @@
+      metadata:
+        name: artists-db
+        description: Stores artist details
+      spec:
+        type: database
+        owner: artist-relations-team
+        system: artist-engagement-portal
+  - name: API
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: API
+      metadata:
+        name: artist-api
+        description: Retrieve artist details
+      spec:
+        type: openapi
+        lifecycle: production
+        owner: artist-relations-team
+        system: artist-engagement-portal
+        definition: |
+          openapi: "3.0.0"
+          info:
+            version: 1.0.0
+            title: Artist API
+            license:
+              name: MIT
+          servers:
+            - url: http://artist.spotify.net/v1
+          paths:
+            /artists:
+              get:
+                summary: List all artists
+  - name: System
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: System
+      metadata:
+        name: artist-engagement-portal
+        description: Handy tools to keep artists in the loop
+      spec:
+        owner: artist-relations-team
+        domain: artists
+  - name: Domain
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: Domain
+      metadata:
+        name: artists
+        description: Everything about artists
+      spec:
+        owner: artist-relations-team
+---
+
+### Install the GitHub App
+
+Installing the GitHub App will allow Roadie to access the YAML metadata files that Backstage needs to operate. Learn more about the [permissions required](/docs/integrations/github-app-permissions/).
+
+1. Click the Administration link in the bottom left of the application.
+
+![A link that says "Administration"](./administration-link.png)
+
+2. Click the Settings tab along the top of the Administration page.
+
+![A link that says "Settings"](./settings-link.png)
+
+3. Click on GitHub under Integrations in the left sidebar.
+
+![A link that says "Integrations"](./integrations-link.png)
+
+4. Open the GitHub accordion and click "Add GitHub App"
+
+![A button that says "Add GitHub App"](./add-github-app.png)
+
+5. Choose your GitHub organization and follow the steps to install the app.
+
+### Upload a catalog item to a repository
+
+In one of the GitHub repos in your project create a `catalog-info.yaml` with the contents shown in the panel on the right.
+
+### Import the catalog file
+
+Copy the URL of the catalog file you created by visiting the file in your browser and copying the url from the location bar. The URL may look like this: `https://github.com/<org-name>/<repo-name>/blob/<branch-name>/catalog-info.yaml`.
+
+Visit the import page in Roadie. `https://<your tenant>.roadie.so/import/entity`, and paste the URL into the box. Click analyze and then import.
+
+![Import](import.png)
+
+Now you can click on the entity link to visit the entity that you have just created.
```

Comment:
Each page should end with Next Steps so the reader is encouraged to keep going and set more and more stuff up. Some good options at the bottom of this page would be:

1. Configure auto-discovery so that Roadie can automatically discover and import `catalog-info.yaml` files. [link to the auto-discovery docs]
2. Use a scaffolder template to give users a streamlined interface that they can use to generate the YAML file you used earlier in this tutorial. [link to here: https://github.com/roadie-demo/getting-started/tree/main/scaffolder/register-new-component]
3. Add a Tech Docs page to the component you now have in the catalog. [link to the next page in the getting started guide] https://deploy-preview-1261--roadie.netlify.app/docs/getting-started/technical-documentation/

Reply:
Ive added some links to the pages now.